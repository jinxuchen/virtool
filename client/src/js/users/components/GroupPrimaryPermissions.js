import React from "react";
import styled from "styled-components";
import { device } from "../../base";
import UserPermissions from "./Permissions";
import UserGroups from "./Groups";
import PrimaryGroup from "./PrimaryGroup";

const StyledGroupPermissions = styled.div`
    @media (min-width: ${device.tablet}) {
        display: grid;
        grid-template-columns: 1fr 1fr;
        grid-gap: 17px;
    }
`;

export const GroupPrimaryPermissions = () => (
    <StyledGroupPermissions>
        <div>
            <UserGroups />

            <PrimaryGroup />
        </div>
        <UserPermissions />
    </StyledGroupPermissions>
);
