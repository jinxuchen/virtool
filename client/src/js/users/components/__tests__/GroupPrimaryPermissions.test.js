import { GroupPrimaryPermissions } from "../GroupPrimaryPermissions";

describe("<GroupPrimaryPermissions />", () => {
    it("should render", () => {
        const wrapper = shallow(<GroupPrimaryPermissions />);
        expect(wrapper).toMatchSnapshot();
    });
});
