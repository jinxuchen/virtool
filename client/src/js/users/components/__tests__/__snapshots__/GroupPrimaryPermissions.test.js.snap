// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GroupPrimaryPermissions /> should render 1`] = `
<GroupPrimaryPermissions__StyledGroupPermissions>
  <div>
    <Connect(UserGroups) />
    <Connect(PrimaryGroup) />
  </div>
  <Connect(UserPermissions) />
</GroupPrimaryPermissions__StyledGroupPermissions>
`;
